gson/src/test/java/com/google/gson/internal/bind/JsonTreeWriterTest.java

```diff
@@ -54,6 +54,27 @@ public void testObject() throws IOException {
     assertEquals("{\"A\":1,\"B\":2}", writer.get().toString());
   }
 
+  public void testJsonValue() throws IOException {
+    JsonTreeWriter writer = new JsonTreeWriter();
+    writer.beginObject();
+    writer.name("int")
+        .jsonValue("1234")
+        .name("double")
+        .jsonValue("12.34")
+        .name("string")
+        .jsonValue("\"hello\"")
+        .name("null")
+        .jsonValue("null")
```

Comment:
Could you please also call `jsonValue(null)`? E.g.:
```suggestion
        .jsonValue("null")
        .name("javaNull")
        .jsonValue(null)
```